### ISSUE TITLE
[Question] How to use custom loss function

### ISSUE THREAD

**Issue by caizp91** [BrikerMan/Kashgari#414]
**You must follow the issue template and provide as much information as possible. otherwise, this issue will be closed.
请按照 issue 模板要求填写信息。如果没有按照 issue 模板填写，将会忽略并关闭这个 issue**

## Check List
Thanks for considering to open an issue. Before you submit your issue, please confirm these boxes are checked.

**You can post pictures, but if specific text or code is required to reproduce the issue, please provide the text in a plain text format for easy copy/paste.**

- [ ] I have searched in [existing issues](https://github.com/BrikerMan/Kashgari/issues?utf8=%E2%9C%93&q=is%3Aissue+) but did not find the same one.
- [ ] I have read the [documents](https://kashgari.bmio.net)

## Environment

- OS [e.g. Mac OS, Linux]:
- Python Version:
- requirements.txt:

```txt
[Paste requirements.txt file here]
```

## Question

[A clear and concise description of what you want to know.]

**Comment by BrikerMan** [comment]
Yes, just via 2 simple steps.

1. After init model, call the `build_model` function with corpus to prepare token dict, label dict, and model structure.
2. Then call the `compile_model` with custom loss, metric, and optimizer (or just one of them).

```python
from kashgari.corpus import SMP2018ECDTCorpus
from kashgari.tasks.classification import BiLSTM_Model
# Remember to import kashgari before than RAdam

train_x, train_y = SMP2018ECDTCorpus.load_data('train')
valid_x, valid_y = SMP2018ECDTCorpus.load_data('valid')
test_x, test_y = SMP2018ECDTCorpus.load_data('test')

model = BiLSTM_Model()

# This step will build token dict, label dict and model structure
model.build_model(train_x, train_y, valid_x, valid_y)

model.compile_model(optimizer="CUSTOM_OPTIMIZER", loss="CUSTOM_LOSS", metrics="CUSTOM_METRICS")

# Train model
model.fit(train_x, train_y, valid_x, valid_y)
```
